lottie/src/main/java/com/airbnb/lottie/model/layer/BaseLayer.java

```diff
@@ -320,16 +321,26 @@ private void intersectBoundsWithMatte(RectF rect, Matrix matrix) {
 
   private void applyMasks(Canvas canvas, Matrix matrix) {
     applyMasks(canvas, matrix, Mask.MaskMode.MaskModeAdd);
+    // Treat intersect masks like add masks. This is not correct but it's closer.
+    applyMasks(canvas, matrix, Mask.MaskMode.MaskModeIntersect);
     applyMasks(canvas, matrix, Mask.MaskMode.MaskModeSubtract);
   }
 
   @SuppressLint("WrongConstant") private void applyMasks(Canvas canvas, Matrix matrix,
       Mask.MaskMode maskMode) {
     Paint paint;
-    if (maskMode == Mask.MaskMode.MaskModeSubtract) {
-      paint = subtractMaskPaint;
-    } else {
-      paint = addMaskPaint;
+    switch (maskMode) {
+      case MaskModeSubtract:
+        paint = subtractMaskPaint;
+        break;
+      case MaskModeIntersect:
+        Log.w(L.TAG, "Animation contains intersect masks. They are not supported but will be " +
+            "treated like intersect masks.");
```

Comment:
It should say: "treated like add masks" :smile:

Reply:
@mister11 Thanks 😛